src/universal/modules/meeting/components/MeetingUpdates/MeetingUpdates.js

```diff
@@ -1,85 +1,103 @@
-import {css} from 'aphrodite-local-styles/no-important';
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, {Component} from 'react';
+import withStyles from 'universal/styles/withStyles';
+import {css} from 'aphrodite-local-styles/no-important';
 import Button from 'universal/components/Button/Button';
-import ProjectColumns from 'universal/components/ProjectColumns/ProjectColumns';
-import MeetingFacilitationHint from 'universal/modules/meeting/components/MeetingFacilitationHint/MeetingFacilitationHint';
 import MeetingMain from 'universal/modules/meeting/components/MeetingMain/MeetingMain';
 import MeetingSection from 'universal/modules/meeting/components/MeetingSection/MeetingSection';
-import actionMeeting from 'universal/modules/meeting/helpers/actionMeeting';
-import getFacilitatorName from 'universal/modules/meeting/helpers/getFacilitatorName';
-import withStyles from 'universal/styles/withStyles';
+import MeetingFacilitationHint from 'universal/modules/meeting/components/MeetingFacilitationHint/MeetingFacilitationHint';
 import {MEETING} from 'universal/utils/constants';
+import ProjectColumns from 'universal/components/ProjectColumns/ProjectColumns';
+import actionMeeting from 'universal/modules/meeting/helpers/actionMeeting';
+import {createFragmentContainer} from 'react-relay';
 
-const MeetingUpdates = (props) => {
-  const {
-    gotoNext,
-    localPhaseItem,
-    members,
-    queryKey,
-    projects,
-    showMoveMeetingControls,
-    styles,
-    team
-  } = props;
-  const self = members.find((m) => m.isSelf);
-  const currentTeamMember = members[localPhaseItem - 1] || {};
-  const isLastMember = localPhaseItem === members.length;
-  const nextPhaseName = actionMeeting.agendaitems.name;
-  const myTeamMemberId = self && self.id;
-  const myUserId = myTeamMemberId && myTeamMemberId.split('::')[0];
-  const isMyMeetingSection = myTeamMemberId === currentTeamMember.id;
+class MeetingUpdates extends Component {
+  state = {};
```

Comment:
We should be declaring the shape of `state`, even if it's initially empty.

Reply:
done!